app/jinja_filters.py

```diff
@@ -546,3 +546,18 @@ def map_list_collector_config(
 @blueprint.app_context_processor
 def map_list_collector_config_processor():
     return dict(map_list_collector_config=map_list_collector_config)
+
+
+@blueprint.app_template_filter()
+def format_paragraphs(text):
+    paragraphs = text.split('\n')
+
+    if len(paragraphs) > 1:
+        return '<p>' + paragraphs.join('</p><p>') + '</p>'
+    else:
```

Comment:
Need to lose this else and de-indent the following line one level.

Reply:
This has been superseded by @jonnyshaw89's suggestion